projects/landing-page/src/App.tsx

```diff
@@ -212,7 +212,9 @@ const App: React.FunctionComponent = () => {
                 <Box>{`await scClient.addWellKnownChain(WellKnownChains.westend);`}</Box>
                 <Box
                   mt={2}
-                >{`const provider = await scClient.addChain(JSON.stringify(parachainSpecs));`}</Box>
+                >{`const provider = await scClient.addWellKnownChain(WellKnownChains.westend2);`}</Box>
```

Comment:
The change should be on the previous line which is still using the old `WellknownChains.wested`, also with this change the `const provider` gets defined twice... So, this line should be removed and the previous line should be edited.